packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/BaseClient.java

```diff
@@ -171,8 +171,7 @@ public abstract class BaseClient {
 		.build();
 	protected Config config;
 	protected Logger logger;
-	CloseableHttpClient httpClient;
-	HttpRequest.Builder httpRequestBuilder;
+	private final HttpClient httpClient;
```

Comment:
Removed `httpRequestBuilder` since it appears to be unused.

Made `httpClient` private as I don't think it was intended to be package-private.